src/ui/Facet/Facet.ts

```diff
@@ -1026,6 +1027,15 @@ export class Facet extends Component {
     return _.map(this.values.getSelected(), (value: FacetValue) => value.value);
   }
 
+  /**
+   * Checks if a single value is selected.
+   * @param value The name of the facet value to verify.
+   */
+  public hasSelectedValue(value: string) {
+    const facetValue = this.values.get(value);
```

Comment:
Why not use `this.values.hasSelectedValue` here similar to line 389 in `DynamicFacet.ts`?

Reply:
It doesn't exist for Facet values. We could always create it though